### ISSUE TITLE
Preview button not pointing to headless client

### ISSUE THREAD

**Issue by MatthewKosloski** [postlight/headless-wp-starter#253]
I installed the `postlight-headless-wp` theme on my local WordPress development environment, and it appears that not all preview buttons are pointing to the headless client.

When navigating to `Pages > All Pages`, hovering over a Draft page and clicking `preview`, the link points to the headless client as expected from the modifications in `functions.php`:

![image](https://user-images.githubusercontent.com/1219553/71301616-f0189280-2366-11ea-9da0-437fbe1bc2d2.png)

As indicated above, that preview "button" properly points to the headless client. 

However...

When navigating to `Pages > All Pages`, clicking on a Draft page, and then on the 'Preview` button on the editor, the link does **not** point to the headless client.

![image](https://user-images.githubusercontent.com/1219553/71301629-32da6a80-2367-11ea-9903-7e45e7e8515a.png)

In other words, the preview link on the "all pages" page has been changed, but the Preview button on the editor view has not.

Is there some additional work I need to do for both links to work? Thanks!

**Comment by modelm** [comment]
This is an upstream bug, Gutenberg ignores the `preview_post_link` filter and offers no alternative at the moment: https://github.com/WordPress/gutenberg/issues/13998
